src/adapter/activateDaffodilDebug.ts

```diff
@@ -85,6 +85,52 @@ async function getFile(fileRequested, label, title) {
   return file
 }
 
+/** Method to select file path to save TDML file
+ * Details:
+ *   Required so that the vscode api commands:
+ *     - extension.dfdl-debug.getValidatedTDMLCopyPath
+ *   can be sent a file instead of always opening up a prompt.
+ */
+async function saveFile(fileRequested, label, title) {
+  let file = ''
+
+  if (fileRequested && fs.existsSync(fileRequested)) {
+    file = fileRequested
+  } else if (fileRequested && !fs.existsSync(fileRequested)) {
+    file = ''
+  } else {
+    file = await vscode.window
```

Comment:
Actually, my suggestion would be to move the `if/else` logic to where this method is called, and have this method be named `showTDMLSaveDialog` and return the file path.

Reply:
We can do that